politeiad/api/v1/v1.go

```diff
@@ -63,7 +64,8 @@ const (
 	RecordStatusCensored          RecordStatusT = 3 // Record has been censored
 	RecordStatusPublic            RecordStatusT = 4 // Record is publicly visible
 	RecordStatusUnreviewedChanges RecordStatusT = 5 // Public visible record that has changes that are not public
-	RecordStatusLocked            RecordStatusT = 6 // Record is locked, note that this has not been implemented yet.
+	RecordStatusReadyForVote      RecordStatusT = 6 // Public visible record that is ready to be voted on
```

Comment:
We really should keep 6 locked. I have actually removed the code for locked and maybe we should get rid of the status altogether. I do like the idea of locking/seal a record forever. So debate me on this. Fix it or toss it?

Ready for vote is a decred specific thing and should be moved into the plugin. Politeia does not understand voting. The ready for vote status is decred-only status.

Reply:
I've thought about it a bit and I also like the idea of being able to lock/seal a record forever.  It explicitly guarantees that the record is preserved as is.  I say we keep it.